src/modm/board/srxe/board.hpp

```diff
@@ -25,21 +27,40 @@ using SystemClock = modm::platform::SystemClock;
 using LedDebug = GpioB0;
 using Leds = SoftwareGpioPort<LedDebug>;
 
-namespace Display {
+namespace spi {
+
+using Sck = GpioB1;
+using Mosi = GpioB2;
+using Miso = GpioB3;
+using SpiMaster = modm::platform::SpiMaster;
+
+}  // namespace spi
+
+namespace DisplayGpio {
 
 using DC = GpioD6;
 using CS = GpioE7;
 using RST = GpioG2;
 
-}  // namespace Display
+}  // namespace DisplayGpio
+
+using Display = modm::St7586s<spi::SpiMaster, DisplayGpio::CS, DisplayGpio::RST, DisplayGpio::DC>;
+extern Display display;
 
 inline void
 initialize() {
 	SystemClock::enable();
 
 	LedDebug::setOutput();
 
+	spi::SpiMaster::connect<spi::Sck::Sck, spi::Mosi::Mosi, spi::Miso::Miso>();
+	spi::SpiMaster::initialize<SystemClock, 1_MHz>();
+	// clock is high when inactive, data is sampled on clock trailing edge
```

Comment:
Something's off either in documentation of DataMode or the description of Mode 3 I found on the Internet. I'll file a separate bug about this.

Reply:
Pushed fix in #674